cmd/parse.go

```diff
@@ -27,3 +32,13 @@ func ParseCommandName(args []string) (string, []string) {
 	}
 	return "", args
 }
+
+// IsTerminal checks if w is a file and whether it is an interactive terminal session.
+func IsTerminal(w io.Writer) bool {
```

Comment:
I think it would be better to just expose this method from the logger package rather than to copy it here.